### PR TITLE
Fix issue with adding Type: Null to a team

### PR DESCRIPTION
**Pull request by HoeenCoder**
Because Type: Null has a colon in its name, it cant be added to teams. This is a fix to it. 

Sorry if i forgot something its my first time editing the client.

### CODE REVIEW

**Comment by HoeenCoder** [comment]
nano isn't working for me (ill fix it tommorow) so i cant squash atm. Sorry about that!

**Comment by Zarel** [comment]
I'm going to recommend just replacing `:` with `|` here

**Comment by HoeenCoder** [comment]
That would work, but how much refactoring would be needed?

**Comment by Zarel** [comment]
not too much

**Comment by kotarou3** [comment]
Now to wait for GF to use `[` and `|` in the game...

**Comment by HoeenCoder** [comment]
Sorry the rebase took so long, but its done now.
Also @kotarou3 thats one of the reasons I used this setup instead of suggestions a refactor.

**Comment by Zarel** [comment]
Holy code duplication, though. Make a temp variable called `entry` if you're going to do it this way.

**Comment by HoeenCoder** [comment]
Sure!

**Comment by Zarel** [comment]
But also I'd still prefer `|`

**Comment by HoeenCoder** [comment]
I could refactor this in the future, but it will take some time. And if GF decides to use `|` were back here.

**Comment by Zarel** [comment]
We could do both.

**Comment by HoeenCoder** [comment]
Just to be clear: refactor from `:` to `|` and keep these edits.

If so should i work on that and push to this PR? Or make a new one and merge this?

**Comment by Zarel** [comment]
Sure.

File: js/client-teambuilder.js [hunk 1]
```diff
@@ -2376,7 +2376,8 @@
 				this.search.find('');
 				return;
 			}
-			var val = $(e.currentTarget).data('entry').split(':')[1];
+			var entry = $(e.currentTarget).data('entry');
+			var val = $(e.currentTarget).data('entry').slice(entry.indexOf(":") + 1);
```
Comment (Zarel) [inline js/client-teambuilder.js]: ...you know, when I said "holy duplicate code", I meant that this should be

    entry.slice(entry.indexOf(":") + 1)

I think you missed the "duplicate code" part.

Comment (HoeenCoder) [inline js/client-teambuilder.js]: Oops my bad

**Comment by Zarel** [comment]
Okay but Dex probably needs to be updated too; lemme see

**Comment by Zarel** [comment]
nvm

**Comment by HoeenCoder** [comment]
One less thing to fix lol

**Pull request merged**